src/publish/mergePublishBranch.ts

```diff
@@ -1,14 +1,23 @@
-import { git } from 'workspace-tools';
+import { git, GitProcessOutput } from 'workspace-tools';
+import { BeachballOptions } from '../types/BeachballOptions';
 
-export function mergePublishBranch(publishBranch: string, branch: string, message: string, cwd: string) {
-  let result: ReturnType<typeof git>;
+export async function mergePublishBranch(
+  publishBranch: string,
+  branch: string,
+  message: string,
+  cwd: string,
+  options: BeachballOptions
+) {
+  let result: GitProcessOutput;
   let mergeSteps = [
     ['add', '.'],
     ['commit', '-m', message],
     ['checkout', branch],
     ['merge', '-X', 'ours', publishBranch],
   ];
 
+  await precommitHook(options);
```

Comment:
Would you mind moving this before the `mergeSteps` and `result`? On initially skimming this I was a bit confused by seeing the commands such as git add before the hook (even though on more careful reading they're not run until after).